i18n/fr/code.json

```diff
@@ -0,0 +1,267 @@
+{
+  "untranslated-page-text": {
+    "message": "This page is not translated. Please, help us translate it with the link at the bottom of the page."
```

Comment:
@Sakrecoer It would be nice to translate this message so that it is displayed in French instead of English.

Reply:
@nop33 Sure thing! Will it adapt if we have more languages?

Reply:
oh, i see what you did!! you are so cool!

Reply:
> Will it adapt if we have more languages?

Yes, given that a translation of this message in the new language is provided.